### DISCUSSION TITLE
[STAC] Load custom collections.json and catalog.json

### DISCUSSION THREAD

**Discussion by iliion** [CS-SI/eodag#413] [Q&A]
Hello,
It is not very clear to me how to "override" the default collections of EODAG. 
I would simply like to use the Stac server and browser with my own collections/catalog.

Could someone give me a hint?
Cheers

**Comment by sbrunato** [comment]
Hello @iliion , the eodag STAC server can be used to "translate" a provider API to a STAC API. You can use EODAG Stac Server with your own catalog if it already has an API:

1. Configure your own server API as a new EODAG provider (see [EODAG documentation / Add a provider](https://eodag.readthedocs.io/en/latest/add_provider.html))
2. Access to your catalog through [EODAG STAC API server](https://eodag.readthedocs.io/en/latest/stac_rest.html)

**Reply by iliion** [reply]
Hum. 
I use the [STAC browser](https://github.com/radiantearth/stac-browser) that i think its also used in EOdag, and I deploy that browser with my own static catalog (`catalog.json` + `collections.json`). There is no API but i was hoping to use EOdag for that. 
Are you telling me there is no (easy) way of doing that?

**Reply by sbrunato** [reply]
You can give [StacStaticSearch](https://eodag.readthedocs.io/en/latest/notebooks/tutos/tuto_stac_client.html#STAC-Static-catalog) plugin a try to set your static catalog as a new provider. But please note that this plugin development is still at an early stage and might be slow if your catalog refers to a huge amount of elements
